lib/base-compiler.js

```diff
@@ -567,10 +567,44 @@ export class BaseCompiler {
         };
     }
 
+    async generateRustMir(inputFilename) {
+        // These arguments make Rustc produce the Rust MIR
+        const execOptions = this.getDefaultExecOptions();
+        // TODO: reconsider this value
+        execOptions.maxOutput = 1024 ** 3;
+        const rustcOptions = [
+            inputFilename,
+            '-o',
+            this.getRustMirOutputFilename(inputFilename),
+            '--emit=mir',
+            '--crate-type',
+            'rlib',
+        ];
```

Comment:
We might want to have some way to change these options? I don't know if other compiler options change the MIR so I hard coded them for now.